distrobox-create

```diff
@@ -292,6 +292,15 @@ generate_command() {
 		--volume /sys:/sys:rslave
 		--volume /tmp:/tmp:rslave"
 
+	# In some systems, for example using sysvinit, /dev/shm is a symlink
+	# to /run/shm, instead of the other way around.
+	# Resolve this detecting if /dev/shm is a symlink and mount original
+	# source also in the container.
+	if [ -L "/dev/shm" ]; then
+		result_command="${result_command}
+			--volume $(realpath /dev/shm)=$(realpath /dev/shm)"
```

Comment:
This needs to be `$(realpath /dev/shm):$(realpath /dev/shm)` I think; once I make that change, everything works (as long as I give it a few seconds between `distrobox-create` and `distrobox-enter`, otherwise I get the same error but I assume that is something weird with WSL2, not `distrobox`).

Reply:
oh yea my bad on this